R/extract_drug_dispenses.R

```diff
@@ -0,0 +1,200 @@
+#' Extraction des délivrances de médicaments.
+#'
+#' Cette fonction permet d'extraire les délivrances de médicaments.
+#' Les délivrances dont les dates EXE_SOI_DTD sont comprises
+#' entre start_date et end_date (incluses) sont extraites.
+#' Le décalage de remontée des données est pris en compte
+#' en récupérant également les délivrances dont les dates
+#' FLX_DIS_DTD sont comprises dans l'année calendaire qui
+#' suit l'année calendaire de end_date.
+#' Si starts_with_codes est fourni, seules les délivrances
+#' de médicaments dont le code ATC commence par l'un des
+#' éléments de starts_with_codes sont extraites.
+#' Si patients_ids est fourni, seules les délivrances
+#' de médicaments pour les patients dont les identifiants
+#' sont dans patients_ids sont extraites.
+#'
+#' @param start_date Date La date de début de la période
+#' des délivrances des médicaments à extraire.
+#' @param end_date Date La date de fin de la période
+#' des délivrances des médicaments à extraire.
+#' @param starts_with_codes Character vector Les codes ATC
+#' par lesquels les délivrances de médicaments à extraire
+#' doivent commencer.
+#' @param patients_ids data.frame Un data.frame contenant les
+#' paires d'identifiants des patients pour lesquels les délivrances de
+#' médicaments doivent être extraites. Les colonnes de ce data.frame
+#' doivent être "ben_idt_ano" et "ben_nir_psa" (en minuscules). Les
+#' "ben_nir_psa" doivent être tous les "ben_nir_psa" associés aux
+#' "ben_idt_ano" fournis.
+#' @return Un data.frame contenant les délivrances de médicaments.
+#' La première colonne est BEN_NIR_PSA si aucun identifiant de
+#' patient n'est fourni. Si  la première colonne est BEN_IDT_ANO.
+#' Les autres colonnes par défaut sont les suivantes :
+#' - EXE_SOI_DTD : Date de la délivrance
+#' - PHA_ACT_QSN : Quantité délivrée
+#' - PHA_ATC_CLA : Code ATC du médicament délivré
+#' - PHA_CIP_C13 : Code CIP du médicament délivré
+#' - PSP_SPE_COD : Code de spécialité du professionnel de soin prescripteur
+#'
+#'
+#' @examples
+#' start_date <- as.Date("2010-01-01")
```

Comment:
Il faut inclure l'exemple dans un  \dontrun{} comme [documenté ici](https://stackoverflow.com/questions/12038160/how-to-not-run-an-example-using-roxygen2#12038225). 

Dans le cas contraire, la construction des pages de documentation R va planter car il essayera de lancer ce code et va se planter car il va essayer d'initier une connection à la base oracle.

Reply:
Pris en compte